js/core.js

```diff
@@ -434,7 +434,7 @@ class PhraseLexer extends Lexer {
             } else if (this.checkEscapeChar('}')) {
                 result += '}';
                 this.advance();
-                this.advance();
+                this.advance();line
```

Comment:
I know this is now merged, but I don't understand this code. What's the line at the end? Just a typo?

Reply:
I think that is what it is :sweat: 
I am sorry, let me just change it.

Reply:
@fdb I have made the correction in #43

Reply:
👍